packages/trader/src/Modules/CFD/Components/cfd-poa.tsx

```diff
@@ -0,0 +1,544 @@
+import { Field, Formik, FieldProps, FormikHelpers as FormikActions, FormikProps, FormikErrors } from 'formik';
```

Comment:
No need to use alias for `FormikHelpers`!

```suggestion
import { Field, Formik, FieldProps, FormikHelpers, FormikProps, FormikErrors } from 'formik';
```

Reply:
Fixed